src/main/java/org/folio/rest/impl/TitlesAPI.java

```diff
@@ -27,8 +54,17 @@ public void getOrdersStorageTitles(String query, String totalRecords, int offset
 
   @Override
   @Validate
-  public void postOrdersStorageTitles(Title entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    PgUtil.post(TITLES_TABLE, entity, okapiHeaders, vertxContext, OrdersStorageTitles.PostOrdersStorageTitlesResponse.class, asyncResultHandler);
+  public void postOrdersStorageTitles(Title title, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    pgClient.withTrans(conn -> titleService.saveTitle(title, conn)
```

Comment:
It's better to use withConn method. Cause with standart isolation level we have got state before starting transaction and it equals queries without transaction

Reply:
Updated